tsschecker/main.c

```diff
@@ -184,7 +186,7 @@ int main(int argc, const char * argv[]) {
         return -1;
     }
 
-    while ((opt = getopt_long(argc, (char* const *)argv, "hd:i:Z:B:e:g:b:u:m:3:8:9:r:c:S:lso0124567p", longopts, &optindex)) > 0) {
+    while ((opt = getopt_long(argc, (char* const *)argv, "hd:i:Z:B:e:g:m:3:8:9:r:c:S:buElso0124567p", longopts, &optindex)) > 0) {
```

Comment:
yah so the ones you removed the colon for need it back. for example -b will do no baseband by default or -b 1
theres also another flag to only request baseband -b 2 Now that there's no colon, optarg isn't inputted so it does default behavior

Reply:
just did it in ae09505, but i'm starting to think, InstallType... wouldn't you do -u 0 or something and it would do Erase?

because if that's the case i could remove -u and -E and replace it with -I for Installtype.
-I 0 would be erase and -I 1 would be update.

Thoughts?

Reply:
i just recompiled tsschecker and tested -E -b and -u

-b 2 does indeed save only baseband tickets, "-u 2" does erase install, and "-E 2" does vice versa. so now i'm stumped.

Now we have description issues, since -u is just said to be "request update tickets only", -E is vice versa.

i can eliminate the optargs for -E and -u, and make them strictly do their respective installtypes so it isn't confusing because what got me was that -E, -u and -b wouldn't do anything since the colons were added, but i didn't know these options had additional functions or how to trigger them.

i also could make a new option called `--install-type` and use `-t` or `-I` for it's short option.
it would replace -u and -E

as for `-b` for `--no-baseband` we should rename it to something like --baseband-opts so it's known that it can change the behavior on how tsschecker saves (or doesn't save) baseband tickets.

i'll wait on your reply so i can get your opinion.

Reply:
@DanTheMann15 My opinion is:
-b yes rename the long name but default behavior is still no baseband and use the mask for other baseband options

I think yes remove E and u so you can do install type 0 erase, 1 update, 2 both erase and update

Reply:
Also probably not a good idea to remove old options, conan doesn't like that kinda thing, maybe you can convince them idk. @DanTheMann15

Reply:
then how about removing the optarg bitmasks from -E and -u so they do just as the program describes it does.

i doubt many realized that -u 2 does erase install, so it's better to just have -u and -E do their respective install types.

Reply:
Fine by me, I recommend erase working by default without -E

Reply:
took care of the bitmasks in 080a36b